pkg/controller/user-cluster-controller-manager/resources/resources/operating-system-manager/rolebinding.go

```diff
@@ -51,6 +52,52 @@ func RoleBindingCreator() reconciling.NamedRoleBindingCreatorGetter {
 	}
 }
 
+// KubePublicRoleBindingCreator returns the RoleBinding for the OSM in kube-system ns.
+func KubePublicRoleBindingCreator() reconciling.NamedRoleBindingCreatorGetter {
+	return func() (string, reconciling.RoleBindingCreator) {
+		return resources.OperatingSystemManagerRoleBindingName, func(rb *rbacv1.RoleBinding) (*rbacv1.RoleBinding, error) {
+			rb.Namespace = metav1.NamespacePublic
+
+			rb.RoleRef = rbacv1.RoleRef{
+				Name:     resources.OperatingSystemManagerRoleName,
+				Kind:     "Role",
+				APIGroup: rbacv1.GroupName,
+			}
+			rb.Subjects = []rbacv1.Subject{
+				{
+					Kind:      rbacv1.ServiceAccountKind,
+					Name:      resources.OperatingSystemManagerRoleBindingName,
+					Namespace: metav1.NamespaceSystem,
+				},
```

Comment:
```suggestion
					Kind:     rbacv1.UserKind,
					Name:     resources.OperatingSystemManagerCertUsername,
					APIGroup: rbacv1.GroupName,
				},
```

Reply:
Are you sure this is a user kind not a service account subject? https://github.com/kubermatic/operating-system-manager/blob/master/deploy/role.yaml#L114-L118

Reply:
Ah ok, yeah that makes sense

Reply:
Sorry I know it's a bit confusing but KKP as a consumer is a special case for OSM. So the sample manifests probably shouldn't cover this.